packages/core/src/App/Containers/Layout/header/trading-hub-onboarding.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import { Icon, Popover } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+
+const TradingHubOnboarding = observer(() => {
```

Comment:
Please rename to `TradersHubOnboarding`.

For now you can rename this and I will create another card to update the rest of them separately.

Reply:
ok, sure.. 👌🏼